src/js/api/core-bundle-loader.js

```diff
@@ -21,7 +21,7 @@ function selectBundle(model) {
     const polyfills = requiresPolyfills();
     const html5Provider = requiresProvider(model, 'html5');
 
-    if (OS.tizen) {
+    if (OS.tizenApp) {
         return loadWebCore();
```

Comment:
I don't think we should change this. Let tizen always load individual modules.

Reply:
mmkay. was debating on this one